File: nan21.dnet.module.pj/nan21.dnet.module.pj.presenter/src/main/java/net/nan21/dnet/module/pj/base/ds/model/IssuePriorityLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2010 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.pj.base.ds.model;

import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;

import net.nan21.dnet.module.pj.base.domain.entity.IssuePriority;
import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.DsField;

@Ds(entity = IssuePriority.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = IssuePriorityLovDs.fSEQUENCENO) })
public class IssuePriorityLovDs extends AbstractTypeLov<IssuePriority> {

    public static final String fSEQUENCENO = "sequenceNo";

    @DsField()
    private Integer sequenceNo;

    public IssuePriorityLovDs() {
        super();
    }

    public IssuePriorityLovDs(IssuePriority e) {
        super(e);
    }

    public Integer getSequenceNo() {
        return this.sequenceNo;
    }

    public void setSequenceNo(Integer sequenceNo) {
        this.sequenceNo = sequenceNo;
    }

}

File: nan21.dnet.module.pj/nan21.dnet.module.pj.ui.extjs/src/main/webapp/src/ds/IssuePriorityLovDs.js
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */   	 
Ext.define("net.nan21.dnet.module.pj.base.ds.model.IssuePriorityLovDs" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"clientId", type:"int", useNull:true},
		{name:"name", type:"string"},
		{name:"active", type:"boolean"},
		{name:"sequenceNo", type:"int", useNull:true}
	]
});

Ext.define("net.nan21.dnet.module.pj.base.ds.model.IssuePriorityLovDsFilter" ,{
	extend: 'Ext.data.Model',
	fields: [ 
		{name:"id", type:"int", useNull:true},
		{name:"name", type:"string"},
		{name:"active", type:"boolean", useNull:true},
		{name:"sequenceNo", type:"int", useNull:true},
		{name:"sequenceNo_From",type:"int", useNull:true},
		{name:"sequenceNo_To",type:"int", useNull:true}
	] 
});
	
Ext.define("net.nan21.dnet.module.pj.base.ds.param.IssuePriorityLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 